.github/workflows/pull-request.yml

```
@@ -0,0 +1,451 @@
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Run Python script
+        run: |
+          python .github/workflows/scripts/code_coverage_disable_check.py --files ${{ steps.changed-files.outputs.all_changed_files }} 
+
+  Test-Application:
+    name: Test Application
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks, Check-ESlint-Disable,Check-Code-Coverage-Disable]
```

YAMLlint (1.35.1)

[warning] too few spaces after comma

(commas)